src/MyElefant.php

```diff
@@ -64,7 +81,8 @@ public function __construct($config)
             );
             throw new Exception(MyElefantConfig::CRITICAL_MESSAGE_EMPTY_SECRET_KEY);
         } else {
-            $this->token = $this->setAuthentification($config['secretKey']);
+            $this->secretKey = $config['secretKey'];
+            $this->token     = 'ZmRmZmZkNzYtNTAwMy00ZmE2LTgwYjItN2QzNzdkYTlhNGYw'; // TODO Change that
```

Comment:
you might want to use config to pass that kind of stuff

Reply:
I forgot to change that.
I have to store the secret key to be able to regenerate the token in the sendSms method